docs/characters.md

```diff
@@ -614,27 +551,26 @@ A bitmask that contains visible actionbars for the player
 | 4    | 0x00000004 | Rigth Bar        |
 | 8    | 0x00000008 | Right Bar 2      |
 
-
 ### grantableLevels
 
 Recruit A Friend stuff.
 
+### order
+
+A field used to change the order in which the characters appear in the character selection screen. The `order` field is used first, then the `[charactesr.guid](charactesr#guid)`, which means that if the `order` column is `NULL` for every character of an account, they will be sorted by `[charactesr.guid](charactesr#guid)` by default.
```

Comment:
You have `charactesr` instead of `characters` in your links

Reply:
fix ok